src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -40,7 +40,11 @@ public class CommandLineTests : CommandLineTestBase
             Path.GetDirectoryName(typeof(CommandLineTests).GetTypeInfo().Assembly.Location),
             Path.Combine("dependency", "csc.exe"));
 
-        private static readonly string s_compilerVersion = typeof(CommandLineTests).Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;
+        private static readonly string s_compilerVersionWithoutHash =
+            typeof(CommandLineTests).Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion.Split('+')[0];
```

Comment:
The `+SHA` is optional (in general). We should have it because we build with Source Link, but there might be some cases when it's not available (e.g. source build, or build on ARM currently). It would be better to to handle cases when there is no `+`.

Reply:
That should work fine. `.Split('+')[0]` returns the full string when there is no `+`.

---
In reply to: [281836290](https://github.com/dotnet/roslyn/pull/35543#discussion_r281836290) [](ancestors = 281836290)

Reply:
Oh, you're right. Cool.